tests/bin/initialize.sh

```diff
@@ -2,3 +2,4 @@
 npm run env run tests-wordpress chmod -- -c ugo+w /var/www/html
 npm run env run tests-cli wp rewrite structure '/%postname%/' -- --hard
 npm run env run tests-cli wp plugin deactivate classic-editor
+npm run env run tests-cli wp plugin activate woocommerce
```

Comment:
Since we only need Woo activated when running a few tests, we should start with it deactivated and then only activate it when we need it (and then deactivate it again after the tests are done)